### DISCUSSION TITLE
posts/webfont

### DISCUSSION THREAD

**Discussion by giscus** [whexy/whexy-blog-comments#19] [General]
# posts/webfont

CS PhD student at Northwestern.

https://www.whexy.com/posts/webfont

**Comment by wangyunzi** [comment]
哇哦这个字体真的很有阅读的感觉，谢谢博主的文章🥰🥰

**Comment by wangyunzi** [comment]
请问博主现在还是使用的这个字体吗？我试用了感觉没效果😭😭

**Reply by whexy** [reply]
不好意思，我忘记更新博客文章，给你添麻烦了🥹。
目前使用的中文字体是 `Noto Sans CJK` （思源黑体，苹果系统里叫 `Source Han Sans`），英文字体是 `Fira Sans`。
这两款字体都是非衬线的。虽然我本人特别喜欢衬线字，但是大多数读者看久了还是会觉得累。最后在衬线字体里找了一个最好看的。

**Reply by wangyunzi** [reply]
哦哦哦好滴好滴，谢谢博主🥰🥰🥰
